web-cheatsheet.adoc

```diff
@@ -8,10 +8,10 @@
 = Main Header
 Optional Author Name <optional@author.email>
 Optional version, optional date
-:Author:    AlternativeWayToSetOptional Author Name
-:Email:     <AlternativeWayToSetOptional@author.email>
-:Date:      AlternativeWayToSetOptional date
-:Revision:  AlternativeWayToSetOptional version
+:author: AlternativeWayToSetOptional Author Name
```

Comment:
Use lowercase for the attribute name and avoid using extra spaces in front of the value.

I'm concerned that the reader will not understand that the format of the author name and email are fixed. It might be better to use concrete examples so they can copy paste.

Reply:
Feel free to change to concrete examples. (Or I can do so later myself.)

Reply:
I recommend using what we use in the official syntax reference:

```
Author Name <author@example.org>
```

Also, it's best to put comments on a line by themselves since AsciiDoc doesn't support trailing line comments. Another option is to use callouts, since those won't be selectable by the mouse.

Reply:
thx, will do!